tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -146,22 +146,6 @@ describe('Header', () => {
     expect(dataFilesTab.hasClass('usa-current')).toEqual(true)
   })
 
-  it("should add usa-current class to dataFiles tab when on '/data-files/*'", () => {
```

Comment:
This test is invalid given our active navitem fix, and also because it makes false assumptions about the app's future state.